### PR TITLE
Updates some sprites, adds missing and new handheld overlays, some fixing, and other stuff.

### PR DESCRIPTION
**Pull request by MrAsbestos**
### Purpose
This PR adds new handheld overlays, fixes various issues, and adds some belt overlays, check it out in the changelog.

### Changelog:
CL: Added a script that makes an error when it detects an empty sprite SO (orginally made by @ThatDan123, thanks Dan!)
CL: Added missing and new handheld first aid kit overlays
CL: Fixed some issues with handheld overlays for drinks
CL: Added some handheld overlays to some snack items
CL: Fixed soda cans being able to be breakable (lol)
CL: Added handheld overlays to basic treatment items
CL: Fixed EVA helmets showing up as syndicate helmets in their handheld overlays
CL: Added missing handheld overlays for donuts
CL: Added the correct handheld overlay for broken bottles
CL: New SM sprite
CL: New Stun Baton sprite
CL: Added missing emergency oxygen tank overlay
CL: Added a sprite for the double-emergency oxygen tank
CL: Added belt overlays for the stun baton, classic baton, flashlight, health analyzer, atmospheric analyzer, and flare
CL: New jukebox sprite, and also added a name, description, price and a centcom message
CL: Added box handheld overlays
CL: Added handheld overlays for normal wrapping paper
CL: Extended the shake distance for all explosions, expect more intense explosions ;)

And that's all folks!

### CODE REVIEW

**Comment by ThatDan123** [comment]
Did you adjust(resize) the SM shield overlay so it'd fit the new sprite size? You'll need to adjust the base one and then the shard variant

**Comment by MrAsbestos** [comment]
Uh, I think I didn't, I tried but I didn't figure it out, and then just kind of forgot about it, and btw, I am not going to be close my PC today, so er don't merge it rn

Was it through the transform settings?

**Comment by MrAsbestos** [comment]
Oh wait, you mean readjust the sprite size via the sprite editor? If you mean that then yes, all new SM sprites (including overlays) were readjusted to their new sizes

File: UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs [hunk 1]
```diff
@@ -47,6 +47,11 @@ private static void DrawProjectItem(string guid, Rect selectionRect)
 			if (Dictionaryguid.ContainsKey(guid))
 			{
 				mainTex = GetCorrectTexture(Dictionaryguid[guid]);
+
+				if (mainTex == null)
+				{
+					Debug.LogError($"Sprite SO {sTing} has null value");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs]: Use this instead
```suggestion
					Logger.LogError($"Sprite SO {sTing} has null value");
```

**Comment by Bod9001** [comment]
"CL: Made the export scanner look less like a gun"
well that's the fun of the export scanner you can make it look like you've got a gun

**Comment by MrAsbestos** [comment]
>  "CL: Made the export scanner look less like a gun"
> well that's the fun of the export scanner you can make it look like you've got a gun

I'l revert that later, don't worry, the funny the shenanigans of sec shooting up a cargo man for mistaking an export scanner for a gun will come back, lolz

File: UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs (hunk 1 above)
Comment (MrAsbestos) [reply UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs]: Done, going to do some other changes, and I'll commit

**Comment by MrAsbestos** [comment]
After this check passes, this should be ready to merge!

**Comment by ThatDan123** [comment]
Missing reference found in prefab SuperMatter component Sprite field Sprite
Missing reference found in prefab SuperMatterHugBox component Sprite field Sprite

probably changed the sprite not the SO on the sprite handler

**Comment by MrAsbestos** [comment]
Oh oops, I'll fix it

**Comment by MrAsbestos** [comment]
This should fix it

**Pull request merged**